project.clj

```diff
@@ -25,13 +25,13 @@
                  [javax.servlet/javax.servlet-api "3.1.0"]
 
                  ;; Jetty Webserver
-                 [org.eclipse.jetty/jetty-server "9.2.8.v20150217"
+                 [org.eclipse.jetty/jetty-server "9.2.10.v20150310"
                   :exclusions [org.eclipse.jetty.orbit/javax.servlet]]
-                 [org.eclipse.jetty/jetty-servlet "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-servlets "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-webapp "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-proxy "9.2.8.v20150217"]
-                 [org.eclipse.jetty/jetty-jmx "9.2.8.v20150217"]
+                 [org.eclipse.jetty/jetty-servlet "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-servlets "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-webapp "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-proxy "9.2.10.v20150310"]
+                 [org.eclipse.jetty/jetty-jmx "9.2.10.v20150310"]
```

Comment:
Would it be worthwhile to `def` this version string? They're all the same, and all have to change at the same time right?